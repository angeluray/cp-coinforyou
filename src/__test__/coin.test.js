import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MyCoin from '../components/coin/coin';
import store from '../redux/configureStore';

it('Test MyCoin renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <MyCoin />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
