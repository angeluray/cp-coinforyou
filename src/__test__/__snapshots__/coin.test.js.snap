// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HomeContainer renders correctly 1`] = `
<div
  className="cryptoContainer"
>
  <input
    className="form"
    onChange={[Function]}
    placeholder="Search a cryptocurrency"
    type="text"
  />
  <h3
    className="stats-header"
  >
    STATS BY RANK
  </h3>
  <ul
    className="mysuperlist"
  />
</div>
`;

exports[`Test MyCoin renders correctly 1`] = `
<div
  className="myCoinContainer"
>
  <div
    className="coinTitle"
  >
    <h2 />
    <p />
  </div>
  <div
    className="myloader"
  >
    <p>
      Counting coins
    </p>
    <img
      alt="loader"
      className="loader"
      src="loading.gif"
    />
  </div>
</div>
`;
